locales-pending/emails-all.ftl

```diff
@@ -0,0 +1,69 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Strings in this file are not yet final, and thus should not be localised yet.
+
+email-header-logo-alt = { -brand-mozilla-monitor }
+email-header-button-sign-in = Sign in
+
+email-footer-support-heading = Questions about { -brand-mozilla-monitor }?
+email-footer-support-content = Visit our <support-link>Support Center</support-link> for help
+# TODO: Check with flod - do we need a token for "Mozilla Support"?
```

Comment:
token = term? No ;-)

Reply:
Yeah like `{ -brand-mozilla-support }`. Thanks for the answer :) 2f825a8a1